$MirOS: ports/mailnews/pine/patches/patch-doc_tech-notes_txt,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
$OpenBSD: patch-doc_tech-notes_txt,v 1.3 2003/05/30 20:44:34 brad Exp $
--- doc/tech-notes.txt.orig	Thu Apr 28 18:29:24 2005
+++ doc/tech-notes.txt	Wed May  4 23:28:07 2005
@@ -2,6 +2,8 @@
                              Pine Technical Notes
                                        
    Version 4.63, April 2005
+   Local version for $MirOS: ports/mailnews/pine/patches/patch-doc_tech-notes_txt,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
+   Not to be confused with the original UW version!
    
    _Pine_ and _Pico_ are trademarks of the University of Washington.
    No commercial use of these trademarks may be made without prior
@@ -219,11 +221,11 @@ Domain Names
    on the Internet will be able to get email back to you.
    
    On UNIX systems, you can set the domain via the user-domain variable
-   in the _Pine_ configuration file, or rely on the file /etc/hosts which
+   in the _Pine_ configuration file, or rely on the file !!ETC!!/hosts which
    usually sets the name of the local host. While _Pine_ can often
    deliver email without the domain name being properly configured, it is
    best to have this set correctly. Problems can usually be solved by
-   adjusting the system's entry in the /etc/hosts file. The
+   adjusting the system's entry in the !!ETC!!/hosts file. The
    fully-qualified name should be listed before any abbreviations.
    
      128.95.112.99 olive.cac.washington.edu olive
@@ -333,14 +335,14 @@ SMTP and Sendmail
           sendmail-path, smtp-server, and compile-time options. The first
           MTA specified in the following list is used:
           
-         1. _sendmail-path_ in /usr/local/lib/pine.conf.fixed
-         2. _smtp-server_ in /usr/local/pine.conf.fixed
+         1. _sendmail-path_ in !!ETC!!/pine.conf.fixed
+         2. _smtp-server_ in !!ETC!!/pine.conf.fixed
          3. _sendmail-path_ specified on the command line.
          4. _smtp-server_ specified on the command line.
          5. _sendmail-path_ in the user's .pinerc file.
          6. _smtp-server_ in the user's .pinerc file.
-         7. _sendmail-path_ in /usr/local/lib/pine.conf
-         8. _smtp-server_ in /usr/local/pine.conf
+         7. _sendmail-path_ in !!ETC!!/pine.conf
+         8. _smtp-server_ in !!ETC!!/pine.conf
          9. DF_SENDMAIL_PATH defined at compile time.
         10. SENDMAIL and SENDMAILFLAGS defined at compile time.
             
@@ -664,20 +666,20 @@ Compile-time Options
    SENDMAIL
    SENDMAILFLAGS
           Sets the name and flags for the local program that will be
-          called to handle outgoing email. Default is /usr/lib/sendmail
+          called to handle outgoing email. Default is /usr/sbin/sendmail
           -oi -oem -t. See the SMTP and Sendmail section for more
           details.
    SYSTEM_PINERC
           The name of the file which holds _Pine_ configuration
           information for all users on the system. Default on UNIX
-          systems is /usr/local/lib/pine.conf.
+          systems is !!ETC!!/pine.conf.
    SYSTEM_PINERC_FIXED
           The name of the file which holds the same type of information
           as for SYSTEM_PINERC, but only for variables that the
           administrator wants to keep fixed. That is, users are not
           allowed to change variables that are specified in the FIXED
           file. Default on UNIX systems is
-          /usr/local/lib/pine.conf.fixed.
+          !!ETC!!/pine.conf.fixed.
           
    There are a couple of more obscure options which are in the source
    code because a few people have asked for them or because we changed
@@ -819,7 +821,7 @@ Building the Pine Programs
                               EXTRACFLAGS=-qro
                                       
    If you are using OpenSSL but have decided to use a non-standard
-   directory (standard one is /usr/local/ssl) for the location then you
+   directory (standard one is !!ETC!!/ssl) for the location then you
    can cause _Pine_ to use that directory with
    
                            SSLDIR=/some/other/dir
@@ -871,24 +873,24 @@ Installing Pine and Pico on UNIX Platfor
    Installing _Pine_ and _Pico_ is remarkably simple. You take the
    program files which you have just transferred or built and you move
    them to the correct directory on your system. Most often the binaries
-   go in /usr/local/bin though sometimes they are placed in /usr/bin. All
-   the help text is compiled into _Pine_ so there are no _required_
+   go in !!PREFIX!!/bin though sometimes they are placed in /usr/bin.
+   All the help text is compiled into _Pine_ so there are no _required_
    auxiliary files.
    
    There are, however, three optional auxiliary files:
-   /usr/local/lib/pine.info, /usr/local/lib/pine.conf, and
-   /usr/local/lib/pine.conf.fixed. The file pine.info contains text on
-   how to get further help on the local system. It is part of the help
-   text for the main menu and should probably refer to the local help
-   desk or the system administrator. If this file doesn't exist a generic
-   version which suggests ``talking to the computer support staff at your
-   site'' is shown. The file pine.conf is used to set system-wide default
-   configurations for _Pine_. The file pine.conf.fixed is also used to
-   set system-wide default configurations for _Pine_. The difference
-   between these two files is that configuration variables set in the
-   pine.conf.fixed file may not normally be over-ridden by a user. See
-   the section on Pine Configuration later in this document for details
-   about the pine.conf and pine.conf.fixed files.
+   !!ETC!!/pine.info, !!ETC!!/pine.conf, and !!ETC!!/pine.conf.fixed. The file
+   pine.info contains text on how to get further help on the local system.
+   It is part of the help text for the main menu and should probably
+   refer to the local help desk or the system administrator. If this file
+   doesn't exist a generic version which suggests ``talking to the
+   computer support staff at your site'' is shown. The file pine.conf is
+   used to set system-wide default configuration for _Pine_. The file
+   pine.conf.fixed is also used to set system-wide default configurations
+   for _Pine_. The difference between these two files is that
+   configuration variables set in the pine.conf.fixed file may not
+   normally be over-ridden by a user. See the section on Pine
+   Configuration later in this document for details about the pine.conf
+   and pine.conf.fixed files.
      _________________________________________________________________
    
 Installing PC-Pine
@@ -941,17 +943,18 @@ Installing IMAPd
 
    When the _Pine_ distribution is built on a UNIX system, the IMAP
    server binary, imapd, is compiled. Installing imapd requires placing
-   the binary in the appropriate directory, usually /usr/etc, and adding
-   entries to /etc/services and /etc/inetd.conf or their counterparts.
-   The following line is appropriate for /etc/services:
+   the binary in the appropriate directory, usually !!PREFIX!!/bin,
+   and adding entries to !!ETC!!/services and !!ETC!!/inetd.conf or their
+   counterparts.
+   The following line is appropriate for !!ETC!!/services:
   imap         143/tcp         # Mail transfer
 
-   and the next line is appropriate for /etc/inetd.conf:
-  imap  stream  tcp     nowait  root    /usr/etc/imapd imapd
+   and the next line is appropriate for !!ETC!!/inetd.conf:
+  imap  stream  tcp     nowait  root    !!PREFIX!!/bin/imapd imapd
 
-   The /etc/inetd.conf file entry may vary on different versions of UNIX.
+   The !!ETC!!/inetd.conf file entry may vary on different versions of UNIX.
    Some have a slightly different set of fields. Also the pathname in
-   /etc/inetd.conf must match the path where imapd is installed.
+   !!ETC!!/inetd.conf must match the path where imapd is installed.
    
    With this configuration, the IMAP server runs without
    pre-authentication. Each new IMAP connection requires a correct
@@ -959,7 +962,7 @@ Installing IMAPd
    based on the standard _rsh_ mechanism. To enable this, the user
    account on the IMAP server must contain a valid file which grants
    access to the client machine. Enabling _rimap_ authentication is done
-   by creating a link called /etc/rimapd to imapd. Basically, what is
+   by creating a link called !!ETC!!/rimapd to imapd. Basically, what is
    happening is that _Pine_ is taking advantage of the ability that rsh
    has to use privileged TCP ports so it doesn't have to run in
    privileged mode. If the rimap authentication fails it will drop back
@@ -976,15 +979,15 @@ Support Files and Environment Variables:
    This section lists the various files which _Pine_ uses which are not
    email folders. All of these are the default names of files, they may
    vary based on _Pine_'s configuration.
-   /usr/local/lib/pine.conf
+   !!ETC!!/pine.conf
           Pine's global configuration file.
-   /usr/local/lib/pine.conf.fixed
+   !!ETC!!/pine.conf.fixed
           Non-overridable global configuration file.
-   /usr/local/lib/pine.info
+   !!ETC!!/pine.info
           Local pointer to system administrator.
-   ~/.pinerc
+   ~/.etc/pinerc
           Personal configuration file for each user.
-   ~/.pinercex
+   ~/.etc/pinercex
           Personal exceptions configuration file for each user.
    ~/.addressbook
           Personal addressbook
@@ -1005,7 +1008,7 @@ Support Files and Environment Variables:
           error which _Pine_ detected.
    ~/mail/postponed-msgs
           A folder of messages which the user chose to postpone.
-   /etc/mailcap
+   !!ETC!!/mailcap
           System-wide mail capabilities file. Only used if $MAILCAPS not
           set.
    ~/.mailcap
@@ -1200,10 +1203,10 @@ Pine
           Configuration: Prints a sample system configuration file to the
           screen or standard output. To generate an initial system
           configuration file, execute
-                pine -conf > /usr/local/lib/pine.conf
+                pine -conf > !!ETC!!/pine.conf
           To generate a system configuration file using settings from an
           old system configuration file, execute
-                pine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+                pine -P old-pine.conf -conf > !!ETC!!/pine.conf
           A system configuration file is not required.
    -convert_sigs _-p pinerc_
           Convert signatures contained in signature files into literal
@@ -1313,14 +1316,14 @@ Pine
           ReadOnly.
    -p _pinerc_
           Uses the named file as the personal configuration file instead
-          of _~/.pinerc_ or the default PINERC search sequence _PC-Pine_
+          of _~/.etc/pinerc_ or the default PINERC search sequence _PC-Pine_
           uses. Pinerc may be either a local file or a remote
           configuration folder.
    -P _pinerc_
           Uses the named file as the system wide configuration file
-          instead of _/usr/local/lib/pine.conf_ on UNIX, or nothing on
-          _PC-Pine_. Pinerc may be either a local file or a remote
-          configuration folder.
+          instead of _!!ETC!!/pine.conf_ on UNIX, or nothing on _PC-Pine_.
+          Pinerc may be either a local file or a remote configuration
+          folder.
    -passfile _passfile_
           This tells _Pine_ what file should be used as the password
           file. This should be a fully-qualified filename.
@@ -1526,13 +1529,13 @@ Pine Configuration
    configuration. In most cases, the compiled-in preferences will suit
    users and administrators just fine. When running _Pine_ on a UNIX
    system, the default built-in configuration can be changed by setting
-   variables in the system configuration files, /usr/local/lib/pine.conf
-   or /usr/local/lib/pine.conf.fixed. (Actually, these files are whatever
+   variables in the system configuration files, !!ETC!!/pine.conf or
+   !!ETC!!/pine.conf.fixed. (Actually, these files are whatever
    the definitions for SYSTEM_PINERC and SYSTEM_PINERC_FIXED in
    pine/osdep/os-xxx.h are set to.) The location of the pine.conf file
    can be changed with the -P command line argument. Both _Pine_ and
    _PC-Pine_ also use personal (user-based) configuration files. On UNIX
-   machines, the personal configuration file is the file ~/.pinerc. For
+   machines, the personal configuration file is the file ~/.etc/pinerc. For
    _PC-Pine_ systems, the personal configuration file is in $PINERC or
    <PineRC registry value> or ${HOME}\PINE\PINERC or <PINE.EXE
    dir>\PINERC. Or the personal configuration file can be specified with
@@ -3167,7 +3170,7 @@ General Configuration Variables
    _personal-name_
           Personal configuration file only. User's full personal name. On
           UNIX systems, the default is taken from the accounts data base
-          (/etc/passwd). The easiest way to change the full From address
+          (!!ETC!!/passwd). The easiest way to change the full From address
           is with the customized-hdrs variable.
    _personal-print-category_
           Personal configuration file only. This is the category that the
@@ -3432,7 +3435,7 @@ General Configuration Variables
           Reverse Color.
    _rsh-command_
           Sets the format of the command used to open a UNIX remote shell
-          connection. The default is "%s %s -l %s exec /etc/r%sd". All
+          connection. The default is "%s %s -l %s exec !!ETC!!/r%sd". All
           four "%s" entries MUST exist in the provided command. The first
           is for the command's pathname, the second is for the host to
           connnect to, the third is for the user to connect as, and the
@@ -3757,7 +3760,7 @@ General Configuration Variables
           spell checker. If your Unix system has _ispell_ it is probably
           reasonable to make it the default speller by configuring it as
           the default in the system configuration file,
-          /usr/local/lib/pine.conf.
+          !!ETC!!/pine.conf.
           If this option is not set, then the system's _spell_ command is
           used. The spell command does not work the same as the alternate
           speller. It produces a list of misspelled words on its standard
@@ -3767,7 +3770,7 @@ General Configuration Variables
           spell command, set the speller option to nothing.
    _ssh-command_
           Sets the format of the command used to open a UNIX secure shell
-          connection. The default is "%s %s -l %s exec /etc/r%sd". All
+          connection. The default is "%s %s -l %s exec !!ETC!!/r%sd". All
           four "%s" entries MUST exist in the provided command. The first
           is for the command's pathname, the second is for the host to
           connnect to, the third is for the user to connect as, and the
@@ -4153,11 +4156,11 @@ General Configuration Variables
           Now for an example:
           
      url-viewers=_TEST("test -n '${DISPLAY}'")_ /usr/local/bin/netscape,
-     /usr/local/bin/lynx, C:\BIN\NETSCAPE.BAT
+     /usr/bin/lynx, C:\BIN\NETSCAPE.BAT
           This example shows that for the first browser in the list to be
           used the environment variable DISPLAY must be defined. If it
           is, then the file /usr/local/bin/netscape must exist. If either
-          condition is not met, then the file /usr/local/bin/lynx must
+          condition is not met, then the file /usr/bin/lynx must
           exist. If it doesn't, then the final path and file must exist.
           Note that the last entry is a DOS/Windows path. This is one way
           to support _Pine_ running on more than one architecture with
@@ -9383,13 +9386,13 @@ Domain Settings
        file
     3. Value of the variable _user-domain_ in the system-wide
        configuration file
-    4. Value from an external database (DNS, /etc/hosts, NIS) as modified
+    4. Value from an external database (DNS, !!ETC!!/hosts, NIS) as modified
        by a system fixed configuration file if use-only-domain-name set
        to _yes_
-    5. Value from an external database (DNS, /etc/hosts, NIS) as modified
+    5. Value from an external database (DNS, !!ETC!!/hosts, NIS) as modified
        by a personal configuration file if _use-only-domain-name_ set to
        _yes_
-    6. Value from an external database (DNS, /etc/hosts, NIS) as modified
+    6. Value from an external database (DNS, !!ETC!!/hosts, NIS) as modified
        by a system configuration file if _use-only-domain-name_ set to
        _yes_
     7. Unmodified value (host name) from an external database
@@ -10084,10 +10087,10 @@ Configuration Inheritance
     5. the system-wide _fixed_ configuration file (Unix _Pine_ only)
        
    The fixed configuration file is normally
-   /usr/local/lib/pine.conf.fixed.
+   !!ETC!!/pine.conf.fixed.
    
    The system-wide configuration file is normally
-   /usr/local/lib/pine.conf for Unix _Pine_ and is normally not set for
+   !!ETC!!/pine.conf for Unix _Pine_ and is normally not set for
    _PC-Pine_. For _PC-Pine_, if the environment variable _$PINECONF_ is
    set, that is used for the system-wide configuration. This location can
    be set or changed on the command line with the -P flag. The
@@ -10095,7 +10098,7 @@ Configuration Inheritance
    configuration folder.
    
    For Unix _Pine_, the personal configuration file is normally the file
-   .pinerc in the user's home directory. This can be changed with the -p
+   .etc/pinerc in the user's home directory. This can be changed with the -p
    command line flag. For _PC-Pine_, the personal configuration file is
    in $PINERC or <PineRC registry value> or ${HOME}\PINE\PINERC or
    <PINE.EXE dir>\PINERC. This can be changed with the -p command line
@@ -10345,19 +10348,18 @@ SMTP Servers
    
    A typical configuration would consist of
      * A program that implements the SMTP or ESMTP protocol via stdio.
-     * An entry in /etc/services for the alternate service.
-     * An entry in /etc/inetd.conf for the alternate service.
-     * An entry in /usr/local/lib/pine.conf,
-       /usr/local/lib/pine.conf.fixed or ~/.pinerc.
+     * An entry in !!ETC!!/services for the alternate service.
+     * An entry in !!ETC!!/inetd.conf for the alternate service.
+     * An entry in !!ETC!!/pine.conf,
+       !!ETC!!/pine.conf.fixed or ~/.etc/pinerc.
      _________________________________________________________________
    
 MIME.Types file
 
    _Pine_'s MIME-TYPE support is based on code contributed by Hans
    Drexler &LT;drexler@mpi.nl&GT;. _Pine_ assigns MIME Content-Types
-   according to file name extensions found in the system-wide files
-   /usr/local/lib/mime.types and /etc/mime.types, and a user specific
-   ~/.mime.types file.
+   according to file name extensions found in the system-wide file
+   !!ETC!!/mime.types, and a user specific ~/.mime.types file.
    
    In DOS and OS/2, _Pine_ looks in the same directory as the PINERC file
    and the same dir as PINE.EXE. This is similar to the UNIX situation
@@ -11336,7 +11338,7 @@ MIME: Reading a Message
    If a MAILCAPSenvironment variable is defined, _Pine_ will use that to
    look for one or more mailcap files, which are combined. In the absence
    of MAILCAPS, Unix _Pine_ will look for a personal mailcap file in
-   ~/.mailcap and combine that with a system-wide file in /etc/mailcap.
+   ~/.mailcap and combine that with a system-wide file in !!ETC!!/mailcap.
    _PC-Pine_ will look for a file named MAILCAP in the same directory as
    the PINERC file, and/or the directory containing the PINE.EXE
    executable.
@@ -11532,7 +11534,7 @@ Printers and Printing
    
    The second selection is the standard UNIX print command. The default
    is _lpr_, but it can be changed on a system basis to anything so
-   desired in /usr/local/lib/pine.conf.
+   desired in !!ETC!!/pine.conf.
    
    The third selection is the user's personal choice for a UNIX print
    command. The text to be printed is piped into the command. _Enscript_
@@ -11986,9 +11988,9 @@ Test Checklist
           Error messages for incorrect terminal types (try "foo" and
           "vt52")
    ___
-          Reading of /usr/local/lib/pine.conf
+          Reading of !!ETC!!/pine.conf
    ___
-          Fixing variables and features in /usr/local/lib/pine.conf.fixed
+          Fixing variables and features in !!ETC!!/pine.conf.fixed
    ___
           Flag command (check message status changed in mail folder)
    ___
